infra/resources.bicep

```diff
@@ -335,6 +325,35 @@ module elasticSearch './modules/container-app.bicep' = {
     ]
     ingressExternal: false
     keyVaultName: keyVault.outputs.AZURE_RESOURCE_KEY_VAULT_NAME
+    initContainersTemplate: [
+      {
+        name: 'init-elasticsearch'
+        image: 'busybox:1.28'
+        resources: {
+          cpu: json('0.25')
+          memory: '0.5Gi'
+        }
+        command: [
+          '/bin/sh'
+        ]
+        args: [
+          '-c'
+          'chown -R 1000:1000 /usr/share/elasticsearch/data && chown -R 1000:1000 /usr/share/elasticsearch/logs'
```

Comment:
it's weird you have to do this, is it because elasticsearch doesn't run as root?

Reply:
That's right
![image](https://github.com/user-attachments/assets/ee8b62e3-cd97-41ba-8260-fbbf017f71c0)